ACKNOWLEDGEMENTS.md

```diff
@@ -3,6 +3,11 @@ Credit for external pieces of work:
 Iris (https://github.com/SciTools/iris), LGPL:
   - .travis.yml, derived from Iris travis setup
   - .gitignore used as basis for ours
+  - environment.yml derived from Iris minimal conda environment
 
 BATS (https://github.com/sstephenson/bats), MIT-style:
   - tests/bin/bats\*, unaltered, from commit 0360811
+
+Automated generation of API documentation from sphinx-build:
+  - Based on solution posted by BowenFu:
+    https://github.com/rtfd/readthedocs.org/issues/1139
```

Comment:
I thought I'd done that - turns out I had, but I'd not added it to the commit - doh!